web/src/test/java/org/openmrs/module/web/WebModuleUtilTest.java

```diff
@@ -256,5 +256,22 @@ private static void setupMocks(boolean includeTrailingSlash) {
 		
 		WebModuleUtil.setDispatcherServlet(dispatcherServlet);
 	}
+	/*
+	startModule should be fixed. ModuleException should not be thrown
+	*/
+	@Test(expected = ModuleException.class)
+	public void startModule_shouldCreateDwrModulesXmlIfNotExists_shouldnotThrowsModuleException() throws Exception {
```

Comment:
Our convention is for each test to have one should instead of two as you have done here. https://wiki.openmrs.org/display/docs/Unit+Testing+With+at-should+Annotation